1-exercises/I-string-replace/exercise.js

```diff
@@ -12,8 +12,13 @@
 
 let story =
   "I like dogs. One day I went to the park and I saw 10 dogs. It was a great day.";
+  
+  const regex = /dogs/g;
+let str = story.replace("10", "100000");
```

Comment:
Method chaining can be used here as the input and output of replace are both strings.